catalog/ui/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -23,9 +23,9 @@ define([
         'usngs',
         'wreqr',
         'js/Common',
-        'backboneassociations'
```

Comment:
Leave this dependency (`backboneassociations`) in, just put it at the end (after `js/CacheSourceSelector`).